test/python/conftest.py

```diff
@@ -1,9 +1,29 @@
 import py
+import os
 import pytest
 import tempfile
 
+from utils import project_name
+from utils import create_project
+
+
 @pytest.fixture(scope='module')
 def module_tmpdir(request):
     dir = py.path.local(tempfile.mkdtemp())
     request.addfinalizer(lambda: dir.remove(rec=1))
     return str(dir)
+
+
+@pytest.fixture(scope="module")
+def project_path(module_tmpdir):
```

Comment:
I suggest you to add finalizer `request.addfinalizer(lambda: shutil.rmtree(path, ignore_errors=True))` to clean directories after test will be finished